### DISCUSSION TITLE
openssl-1.1.1j-win32-mingw.zip and updated curl

### DISCUSSION THREAD

**Discussion by Tarnum-tst** [curl/curl#6630] [Q&A] [answered]
Sorry for newbie question but where i can download openssl-1.1.1j-win32-mingw.zip and updated curl with 1.1.1j? Thanks in advance!

**Comment by bagder** [comment]
It's not provided by us (yet). I'm sure it'll be provided there soon as @vszakats who does almost all the windows packaging is usually very fast with updates.

This said, OpenSSL 1.1.1j fixes just a few security things compared to OpenSSL 1.1.1i and I checked and not a single one of them are used by curl. Therefore, curl users should not have a very strong need for upgrading to this version.

**Reply by Tarnum-tst** [reply]
Thanks very much for the clarification!

**Comment by vszakats** [comment, marked as answer]
Sorry for the delay, I did not notice the new OpenSSL release. [Though I'm subscribed to new OpenSSL releases on GitHub, the release process don't seem to trigger it.]

The new build is [ready](https://github.com/curl/curl-for-win/commit/7f0b2dcfcfa2fe74cec76d09f78db8b8a937b050). It can be downloaded from the [Artifacts](https://ci.appveyor.com/project/curlorg/curl-for-win/builds/37853232/artifacts) tab of the build. It should also appear on the curl download page in a little while.
